eternalcore-core/src/main/java/com/eternalcode/core/translation/implementation/ENTranslation.java

```diff
@@ -417,8 +417,8 @@ public static class ENPrivateSection implements PrivateChatSection {
         public Notice socialSpyMessage = Notice.chat("<dark_gray>[<red>ss<dark_gray>] <dark_gray>[<gray>{SENDER} -> <white>{TARGET}<dark_gray>]<gray>: <white>{MESSAGE}");
 
         @Description(" ")
-        public Notice socialSpyEnable = Notice.chat("<green>► <white>SocialSpy has been <green>enabled!");
-        public Notice socialSpyDisable = Notice.chat("<red>► <white> SocialSpy has been <red>disabled!");
+        public Notice socialSpyEnable = Notice.chat("<green>► <white>SocialSpy has been {STATE}<white>!");
+        public Notice socialSpyDisable = Notice.chat("<red>► <white>SocialSpy has been {STATE}<white>!");
```

Comment:
If you want to replace it with {STATE}, wouldn't it be better to make one notice?
e.g:
`public Notice socialSpyToggled = Notice.chat("<gray>► <white>SocialSpy has been {STATE}<white>!");`

Reply:
this takes away the possibility of notice customization per state

Reply:
As Rollczi said it was made especially, if somebody still would prefer to use custom value in both messages instead of relying on predefined state values